ros2/src/autorccar_costmap/include/costmap.h

```diff
@@ -0,0 +1,82 @@
+#ifndef COSTMAP_H
+#define COSTMAP_H
+
+#include <pcl/common/transforms.h>
+#include <pcl/filters/passthrough.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+
+#include <algorithm>
+#include <eigen3/Eigen/Dense>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+
+struct CostmapInfo {
+    double resolution;    // map resolution [m/cell]
+    unsigned int size_x;  // map width [cell]
+    unsigned int size_y;  // map height [cell]
+    double origin_pos_x;  // real-world pose of the cell (0,0) in the map.
+    double origin_pos_y;
+    Eigen::MatrixXd *costmap;
+};
+
+class Costmap {
+   public:
+    Costmap(const std::string &config_file_path);
```

Comment:
```suggestion
    Costmap(const std::string& config_file_path);
```

구글 가이드라인이 &가 왼쪽에 붙는걸로 알고있는데...?! :open_mouth: